### ISSUE TITLE
I broke Cerebro Plugin Search for Everyone

### ISSUE THREAD

**Issue by codingmatty** [KELiON/cerebro#408]
- [x] I am on the [latest](https://github.com/KELiON/cerebro/releases/latest) Cerebro.app version
- [ ] I have searched the [issues](https://github.com/KELiON/cerebro/issues) of this repo and believe that this is not a duplicate

- OS X 10.12.6
- Cerebro v0.3.1
- Error & Stacktrace:
```
Uncaught (in promise) TypeError: Cannot read property 'replace' of undefined
    at Object.o (main.bundle.js:formatted:7855)
    at Object.c (main.bundle.js:formatted:32302)
    at S.t (main.bundle.js:formatted:16653)
    at h (main.bundle.js:formatted:30850)
    at Function.ii (main.bundle.js:formatted:32873)
    at t (main.bundle.js:formatted:35013)
    at ei (main.bundle.js:formatted:32861)
    at main.bundle.js:formatted:32040
    at m (main.bundle.js:formatted:30862)
    at Sn (main.bundle.js:formatted:32039)
```

## Issue

I published my plugin before creating a github repo, and something about not having a description has broken the `plugins` command altogether.

I have pinpointed it to this line: https://github.com/KELiON/cerebro/blob/f33bce8fc1e4ae23759dacaa4a537c963af97eb0/app/plugins/core/plugins/format.js#L10

**Comment by codingmatty** [comment]
Maybe it's just because I didn't have a `description` in the package.json file.
But either way, I feel like Cerebro should account for that and fail gracefully.

**Comment by codingmatty** [comment]
Oh yea.. I fixed it by republishing my plugin (`cerebro-plugin-convert`)

**Comment by KELiON** [comment]
Ooouch! I'll try to fix it asap

**Comment by codingmatty** [comment]
This seems to have been fixed via PR #409
